### PR TITLE
修改负载均衡章节错误

### PR DESCRIPTION
**Pull request by Toms1999**
image

下面的是conf.d找不到文件

image

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/admin-manual/cluster-management/load-balancing.md [hunk 1]
```diff
@@ -805,4 +805,4 @@ HAProxy是一个使用C语言编写的自由及开放源代码软件，其提供
 
 4. 测试端口是否转换成功
 
-   `mysql -h 172.16.0.3 -uroot -P3307 -p`
\ No newline at end of file
+   `mysql -h 172.16.0.3 -uroot -P3307 -p`
```
Comment (zclllyybb) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/admin-manual/cluster-management/load-balancing.md:808]: what's changed in this line?

**Comment by luzhijing** [comment]
Thanks for your contribution. Please also change the 2.0/dev version docs if the typo exist.

**Review by KassieZ — COMMENTED** [review]
please update 2.1 EN docs and other version(dev / 3.0 / 2.0)

**Comment by KassieZ** [comment]
Thank you for your contribution. However, this pull request contains comments and unresolved conflicts. As a result, it will be closed. If you still wish to submit your changes, please create a new pull request.

**Pull request closed without merging**